privacy-protections/click-to-load/index.html

```diff
@@ -6,33 +6,35 @@
   <title>Facebook click to load</title>
 </head>
 <body>
-  <!-- Load the facebook SDK in the recommended way -->
-  <script async defer crossorigin="anonymous" src="https://connect.facebook.net/en_US/sdk.js?XFBML=false"></script>
+<!-- Load the facebook SDK in the recommended way -->
+<div id="fb-root"></div>
+<script async defer crossorigin="anonymous" src="https://connect.facebook.net/en_US/sdk.js#xfbml=1&version=v14.0" nonce="1SgoaWbh"></script>
 <p><a href="../../">[Home]</a> ↣ <a href="../">[Privacy Protections Tests]</a> ↣ <strong>[Facebook click to load]</strong></p>
 
-<h2> About ClickToLoad Tests</h2>  
-<p>This page embeds the facebook SDK, and several facebook iFrames. It also attempts to create 
-a variety of social elements using the SDK.</P>
+<h2> About ClickToLoad Tests</h2>
+<p>This page embeds the facebook SDK, and several facebook iFrames. It also attempts to create
+a variety of social elements using the SDK.</p>
 
 <p>The page will count how many facebook requests have been sent. In some cases, the browser may start requesting iFrame links, which should be blocked by the extension (but will be counted in the Facebook calls). Calls should not exceed iFrames (and manual verification that those network calls were blocked is a good practice). Once elements are clicked, the count should increase and elements should display (in most cases - this page does not use a valid SDK APP ID, so not all functions work correctly, login in particular.).</p>
 
 <h2> Metrics </h2>
 <div id="facebook_calls"> Facebook Calls: <span id="facebook_call_count">0</span></div>
-<div id="facebook_calls"> Facebook iFrames: <span id="facebook_iFrames">0</span></div>
-<p><button id='download'>Download the result</button></p>
+<div id="facebook_iframes"> Facebook iFrames: <span id="facebook_iFrames">0</span></div>
+<p><button id="download">Download the result</button></p>
 
 <h2> All the social buttons from the SDK </h2>
```

Comment:
Not sure I understand which section you're referring to here?     The section below here?  I think "social" is correct, to be pedantic in FB-speak "share" and "like" aren't the same thing

Reply:
I guess I just mean that perhaps we can simplify this section of the test page. There's two sections that say "All social the buttons..." but then just have one button each. Perhaps we could combine the sections, or change the headings to "Share button..." for example?

![image](https://user-images.githubusercontent.com/155710/178727117-7d532996-3547-44e7-a1c9-87eb0ee039d9.png)

But I don't insist if you think it's OK, just trying to think of ways we can make the page easier to understand. Also I think it is clearer now since you removed the broken example at least.

Reply:
@kzar it should actually look like this (on https://concise-crocus-stetson.glitch.me/)
![image](https://user-images.githubusercontent.com/1753954/178807901-4455af4b-3b63-4420-9292-5e88f1e02246.png)

For FF at least I had to also disable the built-in privacy protections